action_server/src/robocorp/action_server/_settings.py

```diff
@@ -153,3 +154,40 @@ def get_settings() -> Settings:
     if _global_settings is None:
         raise AssertionError("It seems that the settings have not been setup yet.")
     return _global_settings
+
+
+UVICORN_LOG_CONFIG = {
```

Comment:
I think all all log configurations should be done in `robocorp.action_server.cli._setup_stdout_logging` and `robocorp.action_server.cli._setup_logging` (having log configurations scattered in different places makes it harder to reason about it).

i.e.: I'm not sure what's the final result combining the loggers we setup there with the loggers setup here, especially considering the additional handlers added here (stream handlers are added, but what about the handler that redirects to the file, why would `uvicorn.access` no longer be propagated to the file -- is this something we want?)

Reply:
Refactored handling Uvicorn logs – instead of defining Log formatting as suggested by Uvicorn, added a custom Formatter that handles Uvicorn access logs explicitly and a logging filter that does filter out all other Uvicorn messages – these are there if logging level is DEBUG.